### PR TITLE
Add the `COE 1v1 Tournament: Registrations Are Live!` news post and `COE 2024` wiki page

### PR DESCRIPTION
**Pull request by 0x84f**
- [x] make sure yt vid is correct
- [x] confirm banner for news post
- [x] add COE banner for 2024 on the wiki page (if they make one?)
- [x] meyer/cavoe approval (meyer approved via Discord DM)

[preview](https://osu1.roanh.dev/home/news/2024-07-17-coe-x-yukiaim-1v1-tournament)

### CODE REVIEW

File: news/2024/2024-07-17-coe-x-yukiaim-1v1-tournament.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+layout: post
+title: "COE X YUKI AIM 2024 1v1 Tournament: Registrations are live!"
+date: 2024-07-17 20:00:00 +0000
+---
+
+It has never been a better time to showcase your osu skills on LAN than the annual COE 1v1 tournament, now with Yuki Aim as a partner!
+
+![](/wiki/shared/news/2024-07-17-coe-x-yukiaim-1v1-tournament/banner.jpg)
+
+If you are somehow not familiar COE, or it's 1v1 tournament, [COE](/wiki/Community/cavoe%27s_osu%21_event) is osu!'s largest community-hosted event, hosted by [Meyer](https://osu.ppy.sh/users/5452367) and [cavoeboy](https://osu.ppy.sh/users/7361815) from July 30th up until August 4th. Along with the event itself, for the 3rd year in a row the largest LAN tournament of the year will be held there as well, with hundreds of live viewers and even more online on [Twitch](https://www.twitch.tv/COEvent)!
```
Comment (0x84f) [inline news/2024/2024-07-17-coe-x-yukiaim-1v1-tournament.md:11]:
```suggestion
If you are somehow not familiar with COE, or it's 1v1 tournament, [COE](/wiki/Community/cavoe's_osu!_event) is osu!'s largest community-hosted event, hosted by [Meyer](https://osu.ppy.sh/users/5452367) and [cavoeboy](https://osu.ppy.sh/users/7361815) from July 30th up until August 4th. Along with the event itself, for the 3rd year in a row the largest LAN tournament of the year will be held there as well, with hundreds of live viewers and even more online on [Twitch](https://www.twitch.tv/COEvent)!
```

File: news/2024/2024-07-17-coe-x-yukiaim-1v1-tournament.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+| Round of 16 | 2024-08-01 (09:00–21:00) |
+| Quarterfinals | 2024-08-02 (11:30–18:00) |
+| Semifinals | 2024-08-03 (10:30–13:00) |
+| Showmatches | 2024-08-03 (13:30–14:30) |
+| Finals | 2024-08-03 (15:30–16:30) |
+| Grand Finals | 2024-08-03 (18:00–19:30) |
+
+### Prizes
+
+The total prize pool of &euro;3000 will be divided among the top 3 players, along with physical prizes also being handed out.
```
Comment (0x84f) [inline news/2024/2024-07-17-coe-x-yukiaim-1v1-tournament.md:34]:
```suggestion
The total prize pool of €3000 will be divided among the top 3 players, along with physical prizes also being handed out.
```

File: news/2024/2024-07-17-coe-x-yukiaim-1v1-tournament.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+layout: post
+title: "COE X YUKI AIM 2024 1v1 Tournament: Registrations are live!"
+date: 2024-07-17 20:00:00 +0000
+---
+
+It has never been a better time to showcase your osu skills on LAN than the annual COE 1v1 tournament, now with Yuki Aim as a partner!
+
+![](/wiki/shared/news/2024-07-17-coe-x-yukiaim-1v1-tournament/banner.jpg)
+
+If you are somehow not familiar with COE, or it's 1v1 tournament, [COE](/wiki/Community/cavoe's_osu!_event) is osu!'s largest community-hosted event, hosted by [Meyer](https://osu.ppy.sh/users/5452367) and [cavoeboy](https://osu.ppy.sh/users/7361815) from July 30th up until August 4th. Along with the event itself, for the 3rd year in a row the largest LAN tournament of the year will be held there as well, with hundreds of live viewers and even more online on [Twitch](https://www.twitch.tv/COEvent)!
```
Comment (iSlodinxOsu) [inline news/2024/2024-07-17-coe-x-yukiaim-1v1-tournament.md:11]:
```suggestion
If you are somehow not familiar with COE, or it's 1v1 tournament, [COE](/wiki/Community/cavoe's_osu!_event) is osu!'s largest community-hosted event, hosted by [Meyer](https://osu.ppy.sh/users/5452367) & [cavoeboy](https://osu.ppy.sh/users/7361815) from July 30th up until August 4th. Along with the event itself, for the 3rd year in a row the largest LAN tournament of the year will be held there as well, with hundreds of live viewers and even more online on [Twitch](https://www.twitch.tv/COEvent)!
```

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/Community/cavoe's_osu!_event/en.md [hunk 4]
```diff
@@ -8,7 +8,7 @@
 
 ![COE logo](img/logo.png)
 
-[Website](https://cavoeboy.com/) • [Twitter](https://twitter.com/CavoesOsuEvent) • [YouTube](https://www.youtube.com/@coevent) • [Twitch](https://www.twitch.tv/coevent) • [Discord](https://discord.com/invite/d6ru6PVcSY)
+[Website](https://cavoeboy.com/) * [Twitter](https://twitter.com/CavoesOsuEvent) * [YouTube](https://www.youtube.com/@coevent) * [Twitch](https://www.twitch.tv/coevent) * [Discord](https://discord.com/invite/d6ru6PVcSY)
```
Comment (0x84f) [inline wiki/Community/cavoe's_osu!_event/en.md:11]:
```suggestion
[Website](https://cavoeboy.com/) • [Twitter](https://twitter.com/CavoesOsuEvent) • [YouTube](https://www.youtube.com/@coevent) • [Twitch](https://www.twitch.tv/coevent) • [Discord](https://discord.com/invite/d6ru6PVcSY)
```

File: wiki/Community/cavoe's_osu!_event/es.md [hunk 5]
```diff
@@ -8,7 +8,7 @@
 
 ![Logo de COE](img/logo.png)
 
-[Sitio web](https://cavoeboy.com/) • [Twitter](https://twitter.com/CavoesOsuEvent) • [YouTube](https://www.youtube.com/@coevent) • [Twitch](https://www.twitch.tv/coevent) • [Discord](https://discord.com/invite/d6ru6PVcSY)
+[Sitio web](https://cavoeboy.com/) * [Twitter](https://twitter.com/CavoesOsuEvent) * [YouTube](https://www.youtube.com/@coevent) * [Twitch](https://www.twitch.tv/coevent) * [Discord](https://discord.com/invite/d6ru6PVcSY)
```
Comment (0x84f) [inline wiki/Community/cavoe's_osu!_event/es.md:11]:
```suggestion
[Sitio web](https://cavoeboy.com/) • [Twitter](https://twitter.com/CavoesOsuEvent) • [YouTube](https://www.youtube.com/@coevent) • [Twitch](https://www.twitch.tv/coevent) • [Discord](https://discord.com/invite/d6ru6PVcSY)
```

File: wiki/Community/cavoe's_osu!_event/fr.md [hunk 6]
```diff
@@ -8,7 +8,7 @@
 
 ![logo COE](img/logo.png)
 
-[Site web](https://cavoeboy.com/) • [Twitter](https://twitter.com/CavoesOsuEvent) • [YouTube](https://www.youtube.com/@coevent) • [Twitch](https://www.twitch.tv/coevent) • [Discord](https://discord.com/invite/d6ru6PVcSY)
+[Site web](https://cavoeboy.com/) * [Twitter](https://twitter.com/CavoesOsuEvent) * [YouTube](https://www.youtube.com/@coevent) * [Twitch](https://www.twitch.tv/coevent) * [Discord](https://discord.com/invite/d6ru6PVcSY)
```
Comment (0x84f) [inline wiki/Community/cavoe's_osu!_event/fr.md:11]:
```suggestion
[Site web](https://cavoeboy.com/) • [Twitter](https://twitter.com/CavoesOsuEvent) • [YouTube](https://www.youtube.com/@coevent) • [Twitch](https://www.twitch.tv/coevent) • [Discord](https://discord.com/invite/d6ru6PVcSY)
```

File: wiki/Community/cavoe's_osu!_event/en.md
Comment (iSlodinxOsu) [inline wiki/Community/cavoe's_osu!_event/en.md:1]: replace the letter case on chocomint from uppercase to lowercase on both en.md and es.md

**Review by iSlodinxOsu — CHANGES_REQUESTED** [review]

File: wiki/Community/cavoe's_osu!_event/en.md
Comment (0x84f) [reply wiki/Community/cavoe's_osu!_event/en.md:1]: done ty

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2024/2024-07-19-coe-x-yukiaim-1v1-tournament.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+### Prizes
+
+The total prize pool of €3000 will be divided among the top 3 players, along with physical prizes also being handed out.
+
+### Other activities at COE
+
+If you still want to compete, but did not make the top 16, you might be interested in the community-hosted [YokesPai's Extravagant Accuracy Sightread Tournament](https://osu.ppy.sh/community/forums/topics/1944735) accuracy battle royale tournament, focusing on reading and reading-related gimmicks!
+
+COE is also the perfect time to meet your favourite osu! players and content creators, either at the karaoke night, sushi dinner, or just roaming in and around the venue. With familiar faces such as [Spazza17](https://osu.ppy.sh/users/3516241), [Rafis](https://osu.ppy.sh/users/2558286), [Sytho](https://osu.ppy.sh/users/4175698) and [sst3w](https://osu.ppy.sh/users/13337202) already confirmed as attendees, you are sure to find someone you'd love to have a quick chat and take a picture with!
```
[the remark this answers is not in the document]
Comment (0x84f) [reply news/2024/2024-07-19-coe-x-yukiaim-1v1-tournament.md:40]: i wouldnt say its a selling point cuz the pitch has already been done above, but i like how you worded this more so adding it

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: news/2024/2024-07-19-coe-x-yukiaim-1v1-tournament.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "COE X YUKI AIM 2024 1v1 Tournament: Registrations are live!"
```
[the remark this answers is not in the document]
Comment (0x84f) [reply news/2024/2024-07-19-coe-x-yukiaim-1v1-tournament.md:3]: ill change the file name separately but yeah this is fine

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**